R/pip_country_list.R

```diff
@@ -27,6 +28,9 @@ pip_country_list <- function(action = c("update", "load"),
     ## Special national accounts --------
     cl <- pip_country_list_update(class_branch = class_branch)
 
+  # validate country list raw data
+    cl_validate_raw(cl, detail = detail)
```

Comment:
I think i am a little confused. Where does `cl_validate_output()` go? Thanks.

Reply:
The results from this function depends on value of detail argument. By default (detail = FALSE),  for example result cl_validate_output() goes to the console. If detail = TRUE, the result will be emailed (now - will be printed to the console when we run send_report() function)

Thanks